plutus-core/plutus-core/src/PlutusCore/Builtin/KnownType.hs

```diff
@@ -42,6 +40,7 @@ import PlutusCore.Evaluation.Result
 import Control.Monad.Except
 import Data.Coerce
 import Data.DList (DList)
+import Data.Either.Extras
```

Comment:
This is from the plutus-core package, so per our style guide it should be in the previous import group (not that I really care).

This is one reason why I don't like calling it `Data.Either.Extras`.

Reply:
I think our convention says that if it has a `PlutusCore` prefix, it should be in the other import group. If it doesn't have such a prefix, it's meant as something general, so we don't care where it actually lives, it's considered to be a 3rd library function and may eventually become one. @michaelpj correct?

Reply:
The committed style guide actually doesn't mention the grouping of imports because we didn't really reach a conclusion. We probably should actually talk about it. I think I like the idea of just sorting alphabetically like Ziyang proposed. It doesn't matter that much so let's just stick with the simplest thing?